### ISSUE TITLE
Feature: Build multiple tracked rides at once

### ISSUE THREAD

**Issue by ocalhoun6** [OpenRCT2/OpenRCT2#7626]
I can build a path and a tracked ride at the same time, like so:
![forest frontiers 2018-06-03 15-29-05](https://user-images.githubusercontent.com/26573303/40891900-039c464e-6743-11e8-8bfe-d3b3bbc05b9f.png)
And that works great -- they each can be built one piece at a time without interfering with the other. (Though, interestingly, if you hold down shift while doing this, _two_ virtual floors appear.)

But only one ride construction window can be opened at a time. It would be helpful sometimes -- especially when building dueling/racing coasters if it was possible to have both construction windows open at once, instead of having to close one so you can go work on building the other one for a while, then switching back.

Another case where this could be useful is if your'e in the middle of building a big, time-consuming coaster and you decide you need to place a shop or something quickly, you could do so without having to close the ride construction window first, then re-open it.

Now, obviously, this can't work with multiple rides waiting on a mouse click to place the first piece -- but that kind of situation is already handled in the ride/path combination: If you haven't placed the first track piece on the ride yet, opening the path construction window will close the ride construction window and vice versa. I'd imagine that opening multiple ride construction windows at the same time would work the same way -- if you haven't placed the first track piece yet, then opening another ride construction window would close the first one you were working on.

Not sure how feasible/difficult this would be to code ... feel free to ignore it if the way the code is constructed would make multiple ride construction windows a nightmare that required completely changing around the way ride construction works.

**Comment by IntelOrca** [comment]
Currently there are a lot of global variables tied to ride construction so it would require moving all of those to the window struct or waiting until the window struct is inheritable. In order words there are technical reasons why this can't be done at the moment.

**Comment by ocalhoun6** [comment]
Yeah ... kind of suspected that would be the case. Oh well, maybe someday.
